arch/xtensa/src/esp32/esp32_tim.c

```diff
@@ -269,13 +262,10 @@ static int esp32_tim_start(FAR struct esp32_tim_dev_s *dev)
  *
  ****************************************************************************/
 
-static int esp32_tim_stop(FAR struct esp32_tim_dev_s *dev)
+static void esp32_tim_stop(FAR struct esp32_tim_dev_s *dev)
```

Comment:
Ditto, keep is as int

Reply:
Please, note that this stop is from lower half layer not from `esp32_tim.c`. The stop function from lower half is still returning int value.

Reply:
Yes, I noticed it later. But I think it is a good idea to follow the same standard. Also to keep in sync with the other timer drivers.